CHANGELOG.md

```diff
@@ -1,5 +1,19 @@
 # Changelog
 
+## [v0.12.0](https://github.com/googleforgames/agones/tree/v0.12.0) (2019-08-06)
```

Comment:
On the blog page it lists all of the RC changes but here it look like the release doesn't have much different (because they are all captured in the release candidate). Is that how it's been done in the past?

Reply:
Yeah, I tend to copy new features, etc through from RC->Stable for release notes, I figured users will be looking at either and/or stable to see what the new features where.

The changelog is auto generated, and its all in one list, so I figured it was easy to peruse down, so its left as is.

Happy to take other suggestions on how to do this though.